src/main/java/frc/robot/subsystems/swerve/SwerveSubsystem.java

```diff
@@ -305,71 +304,72 @@ public void periodic() {
       // Apply update
       estimator.updateWithTime(sampleTimestamps[deltaIndex], rawGyroRotation, modulePositions);
     }
-    Logger.recordOutput("Swerve Pose", pose);
 
-    for (var camera : cameras) {
-      PhotonPipelineResult result =
-          new PhotonPipelineResult(camera.inputs.latency, camera.inputs.targets);
-      result.setTimestampSeconds(camera.inputs.timestamp);
-      try {
-        var estPose =
+    if (Robot.isReal()) {
```

Comment:
is there a way to contain the sim logic to VisionIOSim?

Reply:
i just changed it but i'm not sure if it makes any sense